e2e/setup.go

```diff
@@ -628,6 +646,53 @@ func (cfg SystemConfig) Start(_opts ...SystemConfigOption) (*System, error) {
 		return nil, fmt.Errorf("unable to start validator: %w", err)
 	}
 
+	// Run validator node (Challenger)
+	challengerCliCfg := validator.CLIConfig{
+		L1EthRpc:                sys.Nodes["l1"].WSEndpoint(),
+		RollupRpc:               sys.RollupNodes["proposer"].HTTPEndpoint(),
+		L2OOAddress:             predeploys.DevL2OutputOracleAddr.String(),
+		ColosseumAddress:        predeploys.DevColosseumAddr.String(),
+		ValPoolAddress:          predeploys.DevValidatorPoolAddr.String(),
+		ChallengerPollInterval:  500 * time.Millisecond,
+		ProverGrpc:              "http://0.0.0.0:0",
+		TxMgrConfig:             newTxMgrConfig(sys.Nodes["l1"].WSEndpoint(), cfg.Secrets.Challenger),
+		OutputSubmitterDisabled: true,
+		SecurityCouncilAddress:  predeploys.DevSecurityCouncilAddr.String(),
+		GuardianEnabled:         true,
+		LogConfig: klog.CLIConfig{
+			Level:  "info",
+			Format: "text",
+		},
+	}
+
+	challengerCfg, err := validator.NewValidatorConfig(challengerCliCfg, sys.cfg.Loggers["challenger"], validatormetrics.NoopMetrics)
+	if err != nil {
+		return nil, fmt.Errorf("unable to init challenger config: %w", err)
+	}
+
+	// Replace to mock RPC client
+	cl, err = rpc.DialHTTP(challengerCliCfg.RollupRpc)
+	if err != nil {
+		return nil, fmt.Errorf("unable to init challenger rollup rpc client: %w", err)
+	}
+	rpcCl = client.NewBaseRPCClient(cl)
+	challengerMaliciousL2RPC := e2eutils.NewChallengerL2RPC(rpcCl)
```

Comment:
Challenger might be honest one, right?
```suggestion
	challengerL2RPC := e2eutils.NewChallengerL2RPC(rpcCl)
```

Reply:
Yes, you're right, we need an honest name.